bot.py

```diff
@@ -131,13 +79,12 @@ async def checkWinStr(reaction,user):
     cur.execute(cmd)
     cmd = "delete from active_matches where win_msg_id = '" + str(reaction.message.id) + "'"
     cur.execute(cmd)
-    conn.commit()
-    cur.close()
+    db_handler.completeTransaction(cur)
     await reaction.message.delete()
-    await leaderboard(reaction.message.channel)
+    await leaderboard.updateLeaderboard(reaction.message.channel, leaderboardMsgs=leaderboardMsgs, leaderboardChannel=leaderboardChannel, db_handler=db_handler)
     
 async def checkStart(message):
```

Comment:
Can we add
if len(message.reactions) < 12:
return

Or would this not be in the refactor change?

Reply:
Added

Reply:
![image](https://user-images.githubusercontent.com/22043313/182657277-8da4a4f6-14bd-4bae-9a6b-8f519db9c386.png)

Reply:
whoops, paren in wrong spot. Fixed.

Reply:
Ok so hear me. Since it is returning "None" hear messes up `if startID != ""` on line 227. We need to return "" here or else it makes it to the startGame line on 228

Reply:
Still doesn't work. Needs to return same thing for line 89. Its returning nothing instead of "". I fixed